.gitignore

```diff
@@ -2,7 +2,7 @@
 .fileStorage/
 .vscode
 .idea
-.env
+.env*
```

Comment:
There's an open issue to add a `.env` file for this repo for local development, and that will involve creating a file named `.env.example` which will need to be committed (same pattern as in other repos). Can you change this ignore rule so that `.env.example` will be committed when we add that?

Reply:
Will do.